src/externs/crypto-js.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @fileoverview crypto-js library extern.
+ */
+
+/** @const */
+var CryptoJS = {};
+
+/**
+ * @param {string} message
+ * @return {string}
```

Comment:
I'm pretty tied repeating my comments, and it is annoying that you didn't even look at my past comments, or check the return value before you reply.

SHA256 returns a compound object on which you need to call toString() to get the hash value, as you did in your code.